Specification/BrAPI-Schema/BrAPI-Core/Study.json

```
@@ -321,7 +319,8 @@
                 "primaryModel": true
             }
         },
-        "EnvironmentParameters": {
+        "EnvironmentParameter": {
+            "description": "An environmental parameter that were kept constant throughout the study and did not change between observation units.\n\nMIAPPE V1.1 (DM-57) Environment - Environmental parameters that were kept constant throughout the study and did not change between observation units or assays. Environment characteristics that vary over time, i.e. environmental variables, should be recorded as Observed Variables.",
```

NIT: Change were to was

```suggestion
            "description": "An environmental parameter that was kept constant throughout the study and did not change between observation units.\n\nMIAPPE V1.1 (DM-57) Environment - Environmental parameters that were kept constant throughout the study and did not change between observation units or assays. Environment characteristics that vary over time, i.e. environmental variables, should be recorded as Observed Variables.",
```

I am curious, why the change from multiple ("EnvironmentParameters") to singular ("EnvironmentParameter") here?

I'm also wondering if including that original description from MIAPPE is redundant, maybe it would just be better served if you just mentioned it was part of MIAPPE, but I can see other descriptions also follow this convention so maybe it is ok.

---

EnvironmentParameters is a schema class. It is used in the Study field experimentalParameters with an array of EnvironmentParameters. In its current form you would a study with List<EnvironmentParameters> experimentalParameters, which is a bit weird. Perhaps @BrapiCoordinatorSelby might know why the schema/class name is plural?